packages/cli/test/dev/fixtures/10a-nextjs-routes/pages/api/env.js

```diff
@@ -0,0 +1,3 @@
+export default (_req, res) => {
+  res.send(process.env);
```

Comment:
This will only test runtime env, not build time env, right?

Reply:
@styfle What's the best way to test build time env?

Reply:
Use the env var on a page

Reply:
Maybe it won't be a problem for `./pages/api/*` because Next.js wont know the difference.

I bet if you used `./api/*` instead then it might make a difference between build/runtime.